python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/metadata.schema.yml

```diff
@@ -34,6 +34,45 @@ keys:
         type: str
         convert_types:
           - int
+      hardware_requirements:
+        type: dict
+        description: |-
+          Defines the minimum hardware specifications required for the device.
+          This section is used for validation and documentation purposes, primarily by the `anta_runner` role.
+        keys:
+          min_power_supplies:
+            type: int
+            description: |-
+              Minimum number of power supplies required for the device.
+              Used for documentation and validation by the `anta_runner` role.
```

Comment:
I don't think you need to repeat for every subfield how it is used (they are all the same)
```suggestion
```